jumanji/env.py

```diff
@@ -70,34 +78,68 @@ def step(self, state: State, action: chex.Array) -> Tuple[State, TimeStep]:
             timestep: TimeStep object corresponding the timestep returned by the environment,
         """
 
-    @abc.abstractmethod
+    @property
     def observation_spec(self) -> specs.Spec:
         """Returns the observation spec.
 
         Returns:
             observation_spec: a NestedSpec tree of spec.
         """
+        return self._observation_spec
 
     @abc.abstractmethod
-    def action_spec(self) -> specs.Spec:
+    def _make_observation_spec(self) -> specs.Spec:
+        """Returns new observation spec.
+
+        Returns:
+            observation_spec: a NestedSpec tree of spec.
```

Comment:
This is unrelated to this PR but all mentions of `NestedSpec` are wrong and come from legacy code. Could we possibly rename to:
```suggestion
            observation_spec: a potentially nested `Spec` structure representing the observation.
```

Reply:
Thank you. I was not aware of this. I will change the references to `NestedSpec`.